sdk/lib/dmilib/dmi.h

```diff
@@ -1,3 +1,12 @@
+/*
+ * PROJECT:     ReactOS DMI/SMBIOS Library
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dmi.h
+ * PURPOSE:     SMBIOS table parsing functions
+ * PROGRAMMERS: Timo Kreuzer (timo.kreuzer@reactos.org)
+ * REFERENCES:  - http://www.dmtf.org/sites/default/files/standards/documents/DSP0134_3.0.0.pdf
```

Comment:
I'd only leave the reference in the .c file, not in the .h
It is not interesting for people using the lib, but it is interesting for people looking inside the lib :)

Reply:
Done!